packages/helm/symphony/values.yaml

```diff
@@ -17,6 +17,12 @@ observability:
   tracing:
     exporter:
       zipkin: true
+  otelCollector:
```

Comment:
Could you add comments for the meaning of newly added parameters?

Reply:
Good point. I can add some comments here.